terraform/base/route53.tf

```diff
@@ -223,6 +223,18 @@ resource "aws_route53_record" "heroku-code-coverage-backend-shipit-cname-test" {
     records = ["coverage.testing.moz.tools.herokudns.com"]
 }
 
+#############################
+##  moz.tools other cnames ##
+#############################
+
+resource "aws_route53_record" "relman-ci-moz-tools-cname-prod" {
+    zone_id = "${aws_route53_zone.moztools.zone_id}"
+    name = "relman-ci.moz.tools"
+    type = "CNAME"
+    ttl = "180"
+    records = ["ec2-35-180-75-156.eu-west-3.compute.amazonaws.com"]
```

Comment:
Is this going to be a stable name for this machine? Should you get an EIP for it?

Reply:
it is a stable name on aws

Reply:
But I don't mind  getting an EIP if you want :)

Reply:
I'm not sure how you have the instance configured but I think a shutdown/restart (eg for AWS maintenance on underlying hardware) can result in a new public IP and broken DNS. AIUI EIP is the usual way to work around that.

Reply:
Sure, I associated the EIP 35.180.7.143 to the server.